velox/dwio/parquet/writer/arrow/Properties.h

```diff
@@ -325,9 +325,7 @@ class PARQUET_EXPORT WriterProperties {
           pagesize_(kDefaultDataPageSize),
           version_(ParquetVersion::PARQUET_2_6),
           data_page_version_(ParquetDataPageVersion::V1),
-          created_by_(
-              std::string("parquet-mr version 2.6.0 (build ") +
-              DEFAULT_CREATED_BY + ")"),
+          created_by_(DEFAULT_CREATED_BY + std::string(" version 0.0.0")),
```

Comment:
Let's add a config option similar to arrow.
Add a code comment: `Velox has no versioning yet. Set default 0.0.0`.